### PR TITLE
Feat/op759/call with value

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
WIP, tracking this in a PR for clarity.

### CODE REVIEW

File: packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol [hunk 1]
```diff
@@ -122,34 +122,7 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
...
-                    "Value could not be transferred to recipient."
-                );
-
-                return (true, bytes(""));
-            } else {
-                // NOTE: Upgrades are temporarily disabled because users can, in theory, modify their EOA
-                // so that they don't have to pay any fees to the sequencer. Function will remain disabled
-                // until a robust solution is in place.
-                require(
-                    transaction.to != Lib_ExecutionManagerWrapper.ovmADDRESS(),
```
Comment (smartcontracts) [inline packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol:147]: I believe this was removed in error. I'm going to add it back in.

File: integration-tests/hardhat.config.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ const config: HardhatUserConfig = {
   },
   solidity: '0.7.6',
   ovm: {
-    solcVersion: '0.7.6',
+    solcVersion: '0.7.6-experimental_callvalue',
```
Comment (snario) [inline integration-tests/hardhat.config.ts:23]: Reminder we have to bump this when preparing for release.

**Comment by tynes** [comment]
@smartcontracts any progress updates on this?

File: packages/contracts/contracts/optimistic-ethereum/libraries/utils/Lib_MathUtils.sol [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/**
+ * @title Lib_MathUtils
+ */
```
Comment (tynes) [inline packages/contracts/contracts/optimistic-ethereum/libraries/utils/Lib_MathUtils.sol:6]: There is a openzeppelin `Math.min` that is already a dependency

**Comment by ben-chain** [comment]
closing in favor of #1038 so I can be updating the status/description/etc as the PR owner.

**Pull request closed without merging**